Changes

```diff
@@ -128,6 +128,9 @@ Next version (4.05.0):
 
 ### Bug fixes
 
+- GPR#934: check for integer overflow in Bytes.extend
+  (Jeremy Yallop)
```

Comment:
Sorting entries by MPR first, GPR second, this should go as the very last item of the category.